pr_check.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Export vars for helper scripts to use
+# --------------------------------------------
+# name of app-sre "application" folder this component lives in; needs to match for quay
+export COMPONENT="acs-ui"
+# IMAGE should match the quay repo set by app.yaml in app-interface
+export IMAGE="quay.io/cloudservices/acs-ui"
+export WORKSPACE=${WORKSPACE:-$APP_ROOT} # if running in jenkins, use the build's workspace
+export APP_ROOT=$(pwd)
+#16 is the default Node version. Change this to override it.
+export NODE_BUILD_VERSION=18
+COMMON_BUILDER=https://raw.githubusercontent.com/RedHatInsights/insights-frontend-builder-common/master
+
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+#TODO (dv) - It isn't clear what these do, what they should be, or if it matters for our purposes.
+IQE_PLUGINS="acs-ui"
+IQE_MARKER_EXPRESSION="smoke_ui"
+IQE_FILTER_EXPRESSION=""
```

Comment:
it looks like this is the differentiating part of the script between this PR check and the build deploy script above -- i wonder what IQE stands for and why we need two different scripts for this 🤔